Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/certificateManager.xhtml

```diff
@@ -471,21 +482,268 @@
+                                                </div>
+                                            </div>
+                                        </div>
+                                        <div class="form-button-row">
+                                            <div class="col-sm-12">
+                                                <p:commandButton value="Copy" ajax="false" icon="ui-icon-clipboard"
+                                                    onclick="document.getElementById('certTabView:importCertTabView:formViewCSR:inputTextareaCsr').select();document.execCommand('copy');alert('copy successful.');return false;" />
+                                                <p:commandButton value="Download " ajax="false" icon="ui-icon-document">
                                                     <p:fileDownload value="#{certificateBean.csrFile}" />
                                                 </p:commandButton>
+                                                <p:commandButton styleClass="delete-button" icon="ui-icon-circle-close"
+                                                    value="Cancel" immediate="true"
+                                                    action="#{certificateBean.cancelImportWizard()}"
+                                                    update="@(.cssImportWizard)">
+                                                    <p:confirm header="Cancel Import Wizard"
+                                                        message="WARNING: This will clean up the file created by wizard. Continue delete?"
+                                                        icon="ui-icon-alert" />
+                                                </p:commandButton>
+                                                <p:commandButton styleClass="btn btn-primary" icon="ui-icon-seek-next"
+                                                    value="Next" action="#{certificateBean.next(3)}"
+                                                    update="@(.cssImportWizard)" immediate="true" />
+
+                                            </div>
+                                        </div>
+                                    </h:form>
+                                </div>
+
+                            </p:tab>
+                            <p:tab id="tabIdCaAuthorityInfo" title="#{certificateBean.tabTitles[3]}" closable="false">
+
+                                <div class="form-block center-block">
+                                    <h:form id="formCaAuthorityInfo" class="form form-horizontal">
+                                        <p:messages id="errorMsgsCaAuthorityInfo" closable="true">
+                                            <p:autoUpdate />
+                                        </p:messages>
+                                        <div class="content">
+                                            <h:outputText>Go to your selected CA Authority website and enter the authorization code and the reference number. 
```

Comment:
Update message like this: "Go to your selected CA Authority website and enter the authorization code and reference number. Follow instructions to generate certificate. Copy the generated certificate  and ??"